src/api/python/cvc5.pxd

```diff
@@ -380,6 +382,29 @@ cdef extern from "api/cpp/cvc5.h" namespace "cvc5::api":
         SortHashFunction() except +
         size_t operator()(const Sort & s) except +
 
+    cdef cppclass Stat:
+        bint isInternal() except +
+        bint isDefault() except +
+        bint isInt() except +
+        int64_t getInt() except +
+        bint isDouble() except +
+        double getDouble() except +
+        bint isString() except +
+        string getString() except +
+        bint isHistogram() except +
+        c_map[string,uint64_t] getHistogram() except +
+
+    cdef cppclass Statistics:
+        Statistics() except +
+        cppclass iterator:
+            iterator() except +
```

Comment:
Is this needed?

Reply:
What exactly do you mean? The whole `Statistics` class?

Reply:
The iterator in the cython cppclass

Reply:
Yes: the list of statistics is exposed via `begin()` and `end()`.